### DISCUSSION TITLE
Built foxglove studio from scratch

### DISCUSSION THREAD

**Discussion by pppaw** [foxglove/community#823] [General]
Is it possible to built complete foxglove studio from scratch? Any resources?

**Comment by jtbandes** [comment]
You can build the open-source application yourself; there are some instructions here: https://github.com/foxglove/studio/blob/main/CONTRIBUTING.md

The application that we offer for download and at studio.foxglove.dev contains some additional features implemented in closed-source code that is not available when building from scratch.

**Comment by ghost** [comment]
hi ! i'm Linux-Scratcher on Scratch :)
